docs(dotnet): TestRunParameters -> Playwright runsettings node (#16505)
microsoft/playwright: docs/src/test-runners-csharp.md

```xml
<?xml version="1.0" encoding="utf-8"?>
<RunSettings>
  <Playwright>
    <Browser>chromium</Browser>

>     <LaunchOptions>

Reviewer: The more a look at this the more it feels that LaunchOptions should be under Browser.

### Using the .runsettings file

When running tests from Visual Studio, you can take advantage of the `.runsettings` file. The following shows a reference of the supported values.

> For example, to specify the amount of workers you can use `MSTest.Parallelize.Workers` or to enable `DEBUG` logs `RunConfiguration.EnvironmentVariables`.

Reviewer 2: ```suggestion
For example, to specify the number of workers, you can use `MSTest.Parallelize.Workers`. You can also enable `DEBUG` logs using `RunConfiguration.EnvironmentVariables`.
```
